packages/zosjobs/src/GetJobs.ts

```diff
@@ -375,7 +375,10 @@ export class GetJobs {
         ImperativeExpect.toNotBeNullOrUndefined(spoolId, "Required parameter spoolId must be defined");
         let parameters: string = "/" + encodeURIComponent(jobname) + "/" + encodeURIComponent(jobid) +
             JobsConstants.RESOURCE_SPOOL_FILES + "/" + encodeURIComponent(spoolId) + JobsConstants.RESOURCE_SPOOL_CONTENT;
-        if (encoding && encoding.trim() != "") {parameters += "?fileEncoding=" + encoding;}
+        if (encoding) {
+            encoding = String(encoding);
+            if(encoding.trim() != "") {parameters += "?fileEncoding=" + encoding;}
```

Comment:
What do you think about combining the two if conditions? 😋 
```suggestion
        if (encoding != null && String(encoding).trim !== "") {
            parameters += "?fileEncoding=" + encoding;
        }
```